initrd/bin/oem-factory-reset

```diff
@@ -520,7 +520,7 @@ generate_OEM_gpg_keys() {
     TRACE "Under oem-factory-reset:generate_OEM_gpg_keys"
 
     #This function simply generates subkeys in smartcard following smarcard config from gpg_key_factory_reset
-    echo "Generating GPG keys in USB Security Dongle' smartcard..."
+    echo "Generating GPG keys in USB Security Dongle's smartcardd..."
```

Comment:
Thanks for catching all the extra missing 's, but this has an extra 'd' now 😢

Reply:
@JonathonHall-Purism forced pushed [8526645](https://github.com/linuxboot/heads/pull/1515/commits/85266452fa932cf1a49b9d1114f56d86a961bb94) to fix this. arrrrg